### DISCUSSION TITLE
Is there a way to insert a link right before the search bar without overriding the navbar component?

### DISCUSSION THREAD

**Discussion by BeeBombshell** [shuding/nextra#2593] [Q&A]
I want to add a link to a form just before the Search bar in the Navbar. Is there a way to do this without overriding the entire navbar component via `navbar.component`?

<img width="1225" alt="image" src="https://github.com/shuding/nextra/assets/53828745/e401a567-1f33-4ec3-812c-47e9f7ad399c">

**Comment by crguezl** [comment]
Same enhancement request than @BeeBombshell. What I believe we need is not to limit the `title` field  in the `_meta.json` file of a folder to be `text` but any kind of JSX/HTML. Something like:

```json
{
  "index": "Introduction",
  "github": {
    "title": "<img src=\"https://github.githubassets.com/assets/GitHub-Mark-ea2971cee799.png\" width=\"50%\" />",
    "type": "page",
    "href": "https://github.com/",
    "newWindow": true
  }
}
